L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h

```diff
@@ -50,21 +66,37 @@ namespace trklet {
     int channelId(int seedType, int layerId) const;
     // max number of seeding layers
     int numSeedingLayers() const { return numSeedingLayers_; }
+    //
+    int nodeDR(const TTTrackRef& ttTrackRef) const;
 
   private:
     // helper class to store configurations
     const tt::Setup* setup_;
-    // use tracklet seed type as channel id if False, binned track pt used if True
-    bool useDuplicateRemoval_;
-    // pt Boundaries in GeV, last boundary is infinity
-    std::vector<double> boundaries_;
+    //
+    int widthLayerId_;
+    //
+    int widthStubId_;
+    //
+    int widthPSTilt_;
+    // positive pt Boundaries in GeV (symmetric negatives are assumed), first boundary is pt cut, last boundary is infinity, defining ot bins used by DR
+    std::vector<double> ptBoundaries_;
+    //
+    int drinDepthMemory_;
+    //
+    int drNumComparisonModules_;
+    //
+    int drMinIdenticalStubs_;
```

Comment:
Can you point me to where this value is set? It should be that tracks are merged if they have 3 or more stubs in common.

Reply:
Here: [https://github.com/cms-L1TK/cmssw/blob/tschuh_dr/L1Trigger/TrackFindingTracklet/python/ChannelAssignment_cfi.py#L12](https://github.com/cms-L1TK/cmssw/blob/tschuh_dr/L1Trigger/TrackFindingTracklet/python/ChannelAssignment_cfi.py#L12)

Reply:
I am happy with the PR if the plan is to use the KF layers in the f/w.

Reply:
@kwalkingshaw if you're happy, then click "Files changed" --> "Review" --> "approve".

Reply:
Can you add in a comment such as "currently set to 3 matching stubs"?

Reply:
done